cpp/tests/replace/replace_tests.cu

```diff
@@ -272,6 +290,63 @@ TYPED_TEST(ReplaceTest, EmptyData)
   }
 }
 
+// Testing with input column containing nulls
+TYPED_TEST(ReplaceTest, NullsInData)
+{
+  this->create_input({7, 5, 6, 3, 1, 2, 8, 4}, {0x0F}, {2, 6, 4, 8}, {0, 4, 2, 6}, {});
```

Comment:
OK since you are updating old tests, but really should use `cudf::test::column_wrapper`.

Reply:
If you suggest, I can update tests to use `cudf::test::column_wrapper`.

Reply:
I think it's fine.